sdk/keyvault/azkeys/autorest.md

```diff
@@ -1,30 +1,123 @@
 ## Go
 
-These settings apply only when `--go` is specified on the command line.
-
 ``` yaml
+clear-output-folder: false
+export-clients: true
 go: true
-version: "^3.0.0"
-input-file:
-- https://github.com/Azure/azure-rest-api-specs/blob/e2ef44b87405b412403ccb005bfb3975411adf60/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/keys.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/ac155b972d0619a6e5bf665a863fb05ee7eeb30f/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/keys.json
 license-header: MICROSOFT_MIT_NO_VERSION
-clear-output-folder: false
-output-folder: internal/generated
+module: github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys
 openapi-type: "data-plane"
+output-folder: ../azkeys
+override-client-name: Client
 security: "AADToken"
 security-scopes:  "https://vault.azure.net/.default"
 use: "@autorest/go@4.0.0-preview.42"
```

Comment:
`preview.43` is the latest code generator.  Maybe update in a separate PR (up to you).